wsd/COOLWSD.cpp

```diff
@@ -4104,6 +4104,12 @@ class COOLWSDServer
            << "\n  UserInterface: " << COOLWSD::UserInterface
             ;
 
+        std::string smap;
+        if (const ssize_t size = FileUtil::readFile("/proc/self/smaps_rollup", smap); size <= 0)
```

Comment:
Some versions of smaps_rollup report incorrect values if rewound back to the start and re-read after the initial read. Doesn't apply here though.